packages/chrome/src/elements/sheet/components/Body.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, HTMLAttributes } from 'react';
+import { StyledSheetBody } from '../../../styled';
+
+export const SheetBody = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  (props, ref) => {
+    return <StyledSheetBody tabIndex={0} ref={ref} {...(props as any)} />;
```

Comment:
It seems like TypeScript compiles without this `as any` type cast. These type casts might be left over from before the `@types/react` was reconciled and can be removed.

Reply:
Could you clarify why the body element can receive keyboard focus?

Reply:
The reason for the `tabIndex=0` is to allow keyboard users to use arrow keys to scroll through the text when it overflows. Is there a better way of doing it?

Reply:
I believe the idiomatic way to achieve that interaction is to set a negative tabindex (`-1`) and programmatically focus on the element when the sheet is opened. Here's an [example](https://yj5ik.csb.app/) of that interaction from a Material UI implementation.

Reply:
Agree with what you are saying. We've removed the `tabIndex` for the time being!